openmoc/plotter.py

```diff
@@ -179,13 +182,13 @@ def plot_segments(track_generator):
   plt.xlim([x.min(), x.max()])
   plt.ylim([y.min(), y.max()])
 
-  title = 'Segments for ' + str(num_azim) + ' angles and ' + str(spacing) + \
-        ' cm spacing'
+  title = 'Segments for ' + str(num_azim) + ' angles, ' + str(spacing) + \
+        ' cm spacing, and ' + str(z[0]) + ' z-level'
```

Comment:
Also, I don't know about the notation you and @geogunow are using for "z levels", but this seems a bit confusing to me. Is the "z[0]" here an integer index for some "z level?" Or is it (as I expect) the z-coordinate for the *xy*-plane plotted here? If it's the latter, perhaps "z-str(z[0])" would be more accurate than "str(z[0])-z-level"?